test/test_main.js

```diff
@@ -87,6 +87,31 @@ function buildMainTests() {
       assert.equal(result.errorMessage, "Test error",
         "Error message should be passed to error display");
     });
+
+    QUnit.test("auto-run functionality structure validation", function (assert) {
```

Comment:
Is this test actually checking anything in main? I suspect that it intended to manipulate the checkbox at `auto-run-check` and validate `_isAutoRunEnabled` has the right values. This could be done by adding elements to `test.html`.